compiler/rustc_lint/src/builtin.rs

```diff
@@ -570,7 +570,12 @@ impl<'tcx> LateLintPass<'tcx> for MissingDoc {
     fn check_crate(&mut self, cx: &LateContext<'_>, krate: &hir::Crate<'_>) {
         self.check_missing_docs_attrs(cx, hir::CRATE_HIR_ID, krate.item.inner, "the", "crate");
 
-        for macro_def in krate.exported_macros {
+        for macro_def in krate.importable_macros {
+            // Non exported MBE 2.0 macros should be skipped
```

Comment:
Please also mention the "why" in this comment

Reply:
Done!